release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -79,6 +79,19 @@ Additionally, SSH access to control plane and compute machines is no longer open
 Installing a cluster on AWS into a secret or top-secret region has not been tested with CAPI as of the release of {product-title} {product-version}. This document will be updated when installation into a secret region has been tested. There is a known issue with Network Load Balancers' support for security groups in secret or top secret regions that causes installations to fail. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-33311[*OCPBUGS-33311*].
 ====
 
+[id="ocp-4-16-installation-and-update-gcp-capi_{context}"]
+==== Cluster API replaces Terraform for {gcp-first} installations (Technology Preview)
+In {product-title} {product-version}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on {gcp-short}. This feature is available as a Technology Preview in {product-title} {product-version}. To enable Technology Preview features, set the `featureSet: TechPreviewNoUpgrade` parameter in the `install-config.yaml` file before installation. Alternatively, add the following stanza to the `install-config.yaml` file before installation to enable Cluster API without any other Technology Preview features:
+
```

Comment:
Do we need the warning about using TP features here?

Reply:
QE requested that I differentiate between the two methods of enabling the feature (all TP features at once, or just this feature).